test/tests/treeview_treeview-1a.js

```diff
@@ -80,6 +80,60 @@ const isClosedFolderTreeitem = async function (el) {
   return (await el.getAttribute('aria-expanded')) === 'false';
 };
 
+ariaTest(
+  'aria-selected attribute on treeitem initial value',
+  exampleFile,
+  'treeitem-aria-selected',
+  async (t) => {
+    const treeitems = await t.context.queryElements(t, ex.treeitemSelector);
+
+    for (let treeitem of treeitems) {
+      t.is(
+        await treeitem.getAttribute('aria-selected'),
+        'false',
+        'treeitem should initially have aria-selected="false"'
+      );
+    }
+  }
+);
+
+ariaTest(
+  'aria-selected attribute on treeitem on down arrow and enter',
```

Comment:
I think this test tries to test too many behaviors in a single test and that one of the behaviors should also be tested with `Key.SPACE`.

So maybe this should be tested with three tests? One to test that selection does not follow focus in response to `Key.DOWN_ARROW`. One to test that `Key.ENTER` selects on each treeitem. One to test that `Key.SPACE` selects on each treeitem.

The 'does not follow focus' test could be something like:

```js
ariaTest(
  'aria-selected attribute does not follow focus',
  exampleFile,
  'treeitem-aria-selected-not-focus',
  async (t) => {
    t.plan((ex.treeitemCount - 1) * 2);
    const items = await t.context.queryElements(t, ex.treeitemSelector);
    // Test all but the last treeitem. The last treeitem does not lose focus when `Key.DOWN_ARROW` is pressed.
    const itemsExcludeLast = items.slice(0, items.length - 1);

    for (let i = 0; i < itemsExcludeLast.length; i++) {
      const treeitem = itemsExcludeLast[i];

      await treeitem.sendKeys(Key.ENTER);

      await treeitem.sendKeys(Key.ARROW_DOWN);
      const focusedItem = await t.context.queryElement(
        t,
        ex.treeitemSelector + '[tabindex="0"]'
      );

      t.is(
        await treeitem.getAttribute('aria-selected'),
        'true',
        'selected treeitem does not change after sending ARROW DOWN'
      );
      t.is(
        await focusedItem.getAttribute('aria-selected'),
        'false',
        'focused treeitem does not have aria-selected="true"'
      );
    }
  }
);
```